tests/all/integration/api/helpers/order/test_calculate_order_amount.py

```diff
@@ -287,3 +391,70 @@ def test_ticket_with_different_discount_code(db):
 
     with pytest.raises(UnprocessableEntity, match='Invalid Discount Code'):
         calculate_order_amount([{'id': ticket.id}], discount.id)
+
+
+def test_request_calculate_order_amount(client, db):
+    discount_code = DiscountCodeTicketSubFactory(type='percent', value=10.0, tickets=[])
+    tickets_dict = _create_taxed_tickets(
+        db, tax_included=False, discount_code=discount_code
+    )
+    db.session.commit()
+
+    response = client.post(
+        '/v1/orders/calculate-amount',
+        content_type='application/json',
+        data=json.dumps(
+            {'tickets': tickets_dict, 'discount-code': str(discount_code.id)}
+        ),
+    )
+
+    assert response.status_code == 200
+    amount_data = json.loads(response.data)
+    assert amount_data['sub_total'] == 4021.87
+    assert amount_data['total'] == 4745.81
+    assert amount_data['tax_included'] is False
+    _assert_tax_data_discount(amount_data)
+
+
+def test_request_calculate_order_amount_without_discount(client, db):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function name "test_request_calculate_order_amount_without_discount" doesn't conform to '[a-z_][a-z0-9_]{1,50}$' pattern](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5528099)